olmo/train.py

```diff
@@ -1263,12 +1269,13 @@ def on_trace_ready(p):
                             python_profiler.print_stats(sort=SortKey.CUMULATIVE)
                             python_profiler = None
                 else:
-                    log.info("Training epoch complete")
+                    log.info(f"Epoch {epoch} complete")
```

Comment:
One thing that may be confusing is that it will say `Epoch 0 complete` and `Epoch 1 complete` will actually mean that 2 epochs have been completed. But fine to leave as is I think.

Reply:
yea true. Actually since we're logging the epoch along with step number I think I'll just revert this.

Reply:
4c7ada39